config/initializers/sentry.rb

```diff
@@ -1,5 +1,8 @@
-Raven.configure do |config|
-  config.async = ->(event) { SentryJob.perform_later(event) }
+Sentry.init do |config|
   config.dsn = ENV['SENTRY_DSN']
-  config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
+  config.breadcrumbs_logger = %i[active_support_logger http_logger]
+  config.environment = Rails.env
+  config.send_default_pii = true
+  config.sample_rate = 1.0
+  config.async = ->(event) { Sentry::SendEventJob.perform_later(event) }
```

Comment:
@jaime-loopstudio Added a couple more configuration options:
```suggestion
  config.async = ->(event) { Sentry::SendEventJob.perform_later(event) }
  config.enabled_environments = %w[production staging] # Specify enabled environments
  # Param filtering
  filter = ActiveSupport::ParameterFilter.new(Rails.application.config.filter_parameters) 
  config.before_send = lambda do |event, hint|
    filter.filter(event.to_hash)
  end
```

Reply:
Thanks for the suggestion! I just added.

Reply:
regarding `enabled_environments`:
As of v0.10.0, events will be sent to Sentry in all environments. If you do not wish to send events in an environment, we suggest you unset the SENTRY_DSN variable in that environment.

so we may not need to set that, and just rely on the dsn being defined or not